docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -418,6 +422,7 @@ Parameter         | Type    | Description
 `pageLoadTimeout` | Number  | Specifies the time (in milliseconds) passed after the `DOMContentLoaded` event, within which TestCafe waits for the `window.load` event to fire. After the timeout passes or the `window.load` event is raised (whichever happens first), TestCafe starts the test. You can set this timeout to `0` to skip waiting for `window.load`. | `3000`
 `speed`           | Number  | Specifies the test execution speed. Should be a number between `1` (the fastest) and `0.01` (the slowest). If speed is also specified for an [individual action](../../test-api/actions/action-options.md#basic-action-options), the action speed setting overrides test speed. | `1`
```

Comment:
A number between `1` (fastest) and `0.01` (slowest). If an [individual action](../../test-api/actions/action-options.md#basic-action-options)'s speed is also specified, the action speed setting overrides the test speed.